theolive/index.mdx

```diff
@@ -17,3 +20,96 @@ We recommend the following steps:
 
 And that's it, you now have a THEOplayer instance that can play the THEOlive source. Now you can interact with the player
 in all kinds of ways, like calling methods and listening to events as explained in this [guide](../theoplayer/getting-started/sdks/web/api-examples).
+
+Welcome to THEOlive! This guide will describe how you can get started with real-time streaming in minutes through the THEOlive management console.
+
+---
+
+THEOlive allows you to stream video real-time to your viewers, and to get started in minutes instead of days or weeks. Leveraging the [High Efficiency Streaming Protocol](https://www.hespalliance.org/), THEOlive aims to provide real-time streaming to large audiences, with high quality video, up to HD 1080p.
+
+THEOlive lives by the "API first" mantra: each action can be performed using API calls. Each possible call can be found at our [API Reference](ref:create-channel). However, to make things easier for non-developers, we built a management console on top of these API calls which makes it straightforward to perform the basic actions. To be able to create and use a THEOlive channel, you should register for an account and get a token to perform [authenticated requests](doc:authentication) to our API.
```

Comment:
Again, `ref:` and `doc:` are specific to Readme.com. Replace them with the correct links:
```suggestion
THEOlive lives by the "API first" mantra: each action can be performed using API calls. Each possible call can be found at our [API Reference](https://developers.theo.live/reference/create-channel). However, to make things easier for non-developers, we built a management console on top of these API calls which makes it straightforward to perform the basic actions. To be able to create and use a THEOlive channel, you should register for an account and get a token to perform [authenticated requests](./getting-started/authentication.mdx) to our API.
```